src/pages/Environments.js

```diff
@@ -409,20 +409,17 @@ export const Environments = ({ nav = undefined }) => {
   }
 
   const renderPauseButton = (computeType, compute) => {
-    const { status } = compute
-    const isAzure = getCloudProvider(compute) === 'AZURE_VM'
     const shouldShowPauseButton = Utils.cond(
       [isApp(compute) && !_.find(tool => tool.appType && tool.appType === compute.appType)(appTools)?.isPauseUnsupported, () => true],
       [isPauseSupported(getToolLabelFromRuntime(compute)) && currentUser === getCreatorForRuntime(compute), () => true],
       () => false)
 
-    return shouldShowPauseButton && h(Link, { //TODO: IA-3993 enable pausing
+    return shouldShowPauseButton && h(Link, {
       style: { marginRight: '1rem' },
-      disabled: isAzure || !isComputePausable(computeType, compute),
-      tooltip: Utils.cond(
-        [isAzure, () => 'Feature coming soon'],
-        [isComputePausable(computeType, compute), () => 'Pause cloud environment'],
-        () => `Cannot pause a cloud environment while in status ${_.upperCase(getComputeStatusForDisplay(status))}.`),
+      disabled: !isComputePausable(computeType, compute),
```

Comment:
Hmm... it it true that Azure Runtimes have the same set of `stopable` statuses as GCE runtimes? 

Leo says `val stoppableStatuses: Set[RuntimeStatus] = Set(Unknown, Running, Updating, Starting)`. Just double checking, cc @mlilynolting

Reply:
Pretty unlikely I'd say - most of our values of that kind are just naive copies of GCP stuff. 

Azure VM statuses are kind of odd, they're concatenated of the last action type and the status associated with the outcome: https://learn.microsoft.com/en-us/rest/api/compute/virtual-machines/instance-view?tabs=HTTP#instanceviewstatus has some examples like
```
  "statuses": [
    {
      "code": "ProvisioningState/succeeded",
      "level": "Info",
      "displayStatus": "Provisioning succeeded",
      "time": "2022-11-01T21:30:12.8051917+00:00"
    },
    {
      "code": "PowerState/running",
      "level": "Info",
      "displayStatus": "VM running"
    }
```

Reply:
Ok, sounds like we sanitize this to a common interface when we get the return from WSM about the status during back leo monitoring

Reply:
In terms of what Leo surfaces to terra-ui though, the answer might be Yes, because Leo mostly seems to treat Azure runtimes as Creating, Running, Stopping, Starting, Deleting... It seems to work for our current system to abstract the platform-specific statuses to this simplified map.